### PR TITLE
sys-kernel/coreos-kernel: depend on coreos-init

### PR DESCRIPTION
**Pull request by bgilbert**
bootengine wants to copy in udev rules it provides.

Fixes https://github.com/coreos/bugs/issues/2531.  Tested on `m5.large`.

### CODE REVIEW

**Comment by bgilbert** [comment]
Travis says:

```
SLOT.invalid [fatal]          2
 coreos-base/coreos-init/coreos-init-0.0.1-r162.ebuild: SLOT: invalid value: '0/0.0.1-r162'
 coreos-base/coreos-init/coreos-init-9999.ebuild: SLOT: invalid value: '0/9999'
```

File: coreos-base/coreos-init/coreos-init-9999.ebuild [hunk 1]
```diff
@@ -21,7 +21,7 @@ HOMEPAGE="http://www.coreos.com/"
 SRC_URI=""
 
 LICENSE="BSD"
-SLOT="0"
+SLOT="0/${PVR}"
```
Comment (dm0-) [inline coreos-base/coreos-init/coreos-init-9999.ebuild]: This needs the ebuild bumped to at least `EAPI=5`.

**Comment by bgilbert** [comment]
Updated.

**Comment by bgilbert** [comment]
Fixed by #3503.

**Pull request merged**